### PR TITLE
Central Component (LandingContent)

### LINKED ISSUE #269: Creating the central component ( Use the same content as shown in Figma)
On our internal team meetings, we have shared the privatized link of Figma Design for our Landing Page. Based on the following, we have to design the page.

Please message on discord, if you need any help :)

### PR DESCRIPTION
**Pull request by syshcndr**
closes #269

### CODE REVIEW

**Review by imabp — APPROVED** [review]
Awesome work @syshcndr Merging to Dev.

File: src/pages/LandingPage/components/LandingContent.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        >
+          <Text pr={2}>Join us on</Text>
+          <FaDiscord /> <Text pl={2}>Discord</Text>
+        </Button>
+      </Link>
+
+      <Text color="white" fontSize="lg" m={4}>
+        Liked our work ? Support us at
+      </Text>
+      <Link href="https://opencollective.com/techphantoms" isExternal>
```
Comment (itsrts) [inline src/pages/LandingPage/components/LandingContent.jsx:41]: we should keep the links in a constant file

File: src/pages/LandingPage/components/LandingContent.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          We are an open source community,
+          <br /> and we build tools for developers and students.
+        </Text>
+        <Text fontSize="md" textAlign="center" m={4}>
+          The thing which makes us unique is, we take our
+          <br /> peer from A Level -{">"} Z Level through.
+          <br /> Learning By Doing.
+        </Text>
+      </Box>
+      <Link href="https://discord.apihacks.co" isExternal>
```
Comment (itsrts) [inline src/pages/LandingPage/components/LandingContent.jsx:25]: same

Comment (syshcndr) [reply src/pages/LandingPage/components/LandingContent.jsx:25]: fixed in #275.
Thank you! 🤩

File: src/pages/LandingPage/components/LandingContent.jsx (hunk 1 above)
Comment (syshcndr) [reply src/pages/LandingPage/components/LandingContent.jsx:41]: fixed in #275.
Thank you! 🤩

**Pull request merged**